src/lib/git.sh

```diff
@@ -0,0 +1,51 @@
+# The git.sh file provides git easy functionalities for kworkflow code.
+
+# Run git bisect
```

Comment:
Please expand this documentation to explain how this function is expected to work and possible pitfalls that a developer may encounter.
P.S.: Review all your documentation for this file. Remember, this is a library file and we want to keep it easy to understand.